pinot-common/src/main/java/org/apache/pinot/common/response/broker/BrokerResponseNative.java

```diff
@@ -277,6 +279,31 @@ public void setTimeUsedMs(long timeUsedMs) {
     _timeUsedMs = timeUsedMs;
   }
 
+  @JsonProperty("offlineThreadCpuTimeNs")
+  @Override
+  public long getOfflineThreadCpuTimeNs() {
+    return _offlineThreadCpuTimeNs;
+  }
+
+  @JsonProperty("offlineThreadCpuTimeNs")
+  @Override
+  public void setOfflineThreadCpuTimeNs(long timeUsedMs) {
```

Comment:
(nit) the argument name should have Ns instead of Ms. In fact, I suggest it to name appropriately -- `offlineThreadCpuTimeNs`

Reply:
done